packages/editor/src/client/blocks/richTextBlock/editor/normalizing.ts

```diff
@@ -0,0 +1,197 @@
+import {Transforms, Element as SlateElement, Node as SlateNode, Path} from 'slate'
+import {ReactEditor} from 'slate-react'
+import {defaultBorderColor, emptyTextParagraph} from './elements'
+import {BlockFormat} from './formats'
+
+// See: https://github.com/ianstormtaylor/slate/blob/master/Changelog.md#0530--december-10-2019
+
+// TODO
+// - TableCell needs Paragraph as child
+
+export function withNormalizeNode<T extends ReactEditor>(editor: T): T {
+  const {normalizeNode} = editor
+
+  editor.normalizeNode = entry => {
+    const [node, path] = entry
+
+    if (SlateElement.isElement(node)) {
+      switch (node.type) {
+        // order of corrections is important !
```

Comment:
Maybe improve this comment with a small but detailed explanation what exactly is happening here.